### PR TITLE
chore(docs): editorial review of transforms and sources

### PR DESCRIPTION
**Pull request by maycmlee**
Editorial review of transforms and sources descriptions to fit our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: src/sources/logstash.rs [hunk 1]
```diff
@@ -51,13 +51,13 @@ pub struct LogstashConfig {
 
     /// The size of the receive buffer used for each connection.
     ///
-    /// This generally should not need to be changed.
+    /// Generally, this should not need to be changed.
```
Comment (maycmlee) [inline src/sources/logstash.rs:54]: When would it need to be changed?

File: src/sources/socket/tcp.rs [hunk 2]
```diff
@@ -72,7 +72,7 @@ pub struct TcpConfig {
 
     /// The size of the receive buffer used for each connection.
     ///
-    /// Generally this should not need to be configured.
+    /// Generally, this should not need to be configured.
```
Comment (maycmlee) [inline src/sources/socket/tcp.rs:75]: When would it need to be changed?

File: src/sources/socket/udp.rs [hunk 3]
```diff
@@ -71,7 +71,7 @@ pub struct UdpConfig {
 
     /// The size of the receive buffer used for the listening socket.
     ///
-    /// Generally this should not need to be configured.
+    /// Generally, this should not need to be configured.
```
Comment (maycmlee) [inline src/sources/socket/udp.rs:74]: When would it need to be changed?

File: src/sources/statsd/mod.rs [hunk 4]
```diff
@@ -69,7 +69,7 @@ pub struct UdpConfig {
 
     /// The size of the receive buffer used for each connection.
     ///
-    /// Generally this should not need to be configured.
+    /// Generally, this should not need to be configured.
```
Comment (maycmlee) [inline src/sources/statsd/mod.rs:72]: When would it need to be changed?

File: src/sources/statsd/mod.rs [hunk 5]
```diff
@@ -104,11 +104,11 @@ pub struct TcpConfig {
...
```
Comment (maycmlee) [inline src/sources/statsd/mod.rs:107]: When would it need to be changed?

File: src/sources/exec/mod.rs [hunk 6]
```diff
@@ -117,7 +117,7 @@ pub struct StreamingConfig {
     #[serde(default = "default_respawn_on_exit")]
     respawn_on_exit: bool,
 
-    /// The amount of time, in seconds, that Vector will wait before rerunning a streaming command that exited.
+    /// The amount of time, in seconds, before rerunning a streaming command that exited.
```
[the remark this answers is not in the document]
Comment (spencergilbert) [reply src/sources/exec/mod.rs:120]: @jszwedko do you mind getting someone to update the types and configurable annotations here, or open an issue to to so?

**Review by spencergilbert — COMMENTED** [review]
Sorry - things came up and I won't be able to finish reviewing this.

File: src/sources/logstash.rs (hunk 1 above)
Comment (jszwedko) [reply src/sources/logstash.rs:54]: Same comment applies to the below ones 😄

**Review by jszwedko — APPROVED** [review]
Awesome, thanks @maycmlee !

What is the best way to make sure this copy and new copy stays up-to-date with our style guide? Should we tag you / your team for review on changes? Or just do our best and review it every once in a while (say once a quarter?)? Or something else?

File: src/sources/socket/tcp.rs (hunk 2 above)
Comment (neuronull) [reply src/sources/socket/tcp.rs:75]: For all of these flagged "Generally, this should not need to be configured."  , all for receive buffer sizes,

, I wonder if we should just remove those from the doc comment altogether.
At one point we had in the external docs, flags for common/uncommon configuration settings, but those don't exist anymore.

File: src/sources/logstash.rs (hunk 1 above)
Comment (neuronull) [reply src/sources/logstash.rs:54]: Ah, I missed that you'd already commented to this effect Jesse in my below reply to one of the other places it was flagged.

Comment (maycmlee) [reply src/sources/logstash.rs:54]: Thanks @jszwedko @neuronull for explaining! You both seem to agree that we could drop the "Generally, this should not need to be changed" statement, so I'll go ahead and remove all instances of where I've flagged it.

File: src/sources/socket/udp.rs (hunk 3 above)
Comment (maycmlee) [reply src/sources/socket/udp.rs]: Removed

File: src/sources/statsd/mod.rs (hunk 5 above)
Comment (maycmlee) [reply src/sources/statsd/mod.rs]: Removed sentence

File: src/sources/statsd/mod.rs (hunk 4 above)
Comment (maycmlee) [reply src/sources/statsd/mod.rs]: Removed sentence

File: src/sources/logstash.rs (hunk 1 above)
Comment (maycmlee) [reply src/sources/logstash.rs]: Removed all

**Comment by maycmlee** [comment]
> Awesome, thanks @maycmlee !
> 
> What is the best way to make sure this copy and new copy stays up-to-date with our style guide? Should we tag you / your team for review on changes? Or just do our best and review it every once in a while (say once a quarter?)? Or something else?

Thanks for reviewing @jszwedko! Yes, tagging the [documentation team](https://github.com/orgs/DataDog/teams/documentation) for review would be best! It'll show up in our on-call PR list and the person on call that day (business hours only) would review it. For for larger reviews, they'd create a docs card and then someone would be assigned to review it. You can also ping in #documentation if you need a quick turnaround.

I've made the changes mentioned in the comments, let me know if I've missed anything!

**Pull request merged**